aptos-move/move-crates/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "move-crates"
+version = "0.0.1"
+authors = ["Aptos Labs <opensource@aptoslabs.com>"]
+repository = "https://github.com/aptos-core/aptos-labs"
+homepage = "https://aptoslabs.com"
+license = "Apache-2.0"
+publish = false
+edition = "2018"
+
+[dependencies]
+move-compiler = { git = "https://github.com/move-language/move", rev = "f2e7585b1ed5bd2810163d6bdebafe5a388881d3" }
```

Comment:
nit: I'm gonna be really annoying and suggest that we keep this list in alphabetical ordering. Especially if we ever have to update it 😄

Reply:
awesome, sir.